### PR TITLE
Cognitive Services: Added additional parameters

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

- Added additional parameters
- Regenerated docs

Pipeline reference:
[CognitiveServices: Accounts](https://github.com/Azure/ResourceModules/actions/workflows/ms.cognitiveservices.accounts.yml)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

File: arm/Microsoft.CognitiveServices/accounts/readme.md [hunk 1]
```diff
@@ -41,17 +43,23 @@ This module deploys different kinds of cognitive services resources
 | `diagnosticSettingsName` | string | `[format('{0}-diagnosticSettings', parameters('name'))]` |  | The name of the diagnostic setting, if deployed. |
 | `diagnosticStorageAccountId` | string | `''` |  | Resource ID of the diagnostic storage account. |
 | `diagnosticWorkspaceId` | string | `''` |  | Resource ID of the diagnostic log analytics workspace. |
+| `disableLocalAuth` | bool | `False` |  | Allow only Azure AD authentication. |
 | `enableDefaultTelemetry` | bool | `True` |  | Enable telemetry via the Customer Usage Attribution ID (GUID). |
+| `encryption` | object | `{object}` |  | Properties to configure Encryption |
```
Comment (eriqua) [inline arm/Microsoft.CognitiveServices/accounts/readme.md:48]:
```suggestion
| `encryption` | object | `{object}` |  | Properties to configure encryption |
```

File: arm/Microsoft.CognitiveServices/accounts/deploy.bicep [hunk 2]
```diff
@@ -106,6 +106,30 @@ param roleAssignments array = []
 @description('Optional. Tags of the resource.')
 param tags object = {}
 
+@description('Optional. List of allowed FQDN.')
+param allowedFqdnList array = []
+
+@description('Optional. The api properties for special APIs.')
```
Comment (eriqua) [inline arm/Microsoft.CognitiveServices/accounts/deploy.bicep:112]:
```suggestion
@description('Optional. The API properties for special APIs.')
```

File: arm/Microsoft.CognitiveServices/accounts/deploy.bicep [hunk 3]
```diff
@@ -106,6 +106,30 @@ param roleAssignments array = []
...
+param apiProperties object = {}
+
+@description('Optional. Allow only Azure AD authentication.')
+param disableLocalAuth bool = false
+
+@description('Optional. Properties to configure Encryption')
```
Comment (eriqua) [inline arm/Microsoft.CognitiveServices/accounts/deploy.bicep:118]:
```suggestion
@description('Optional. Properties to configure encryption')
```

File: arm/Microsoft.CognitiveServices/accounts/deploy.bicep [hunk 4]
```diff
@@ -106,6 +106,30 @@ param roleAssignments array = []
...
+param encryption object = {}
+
+@description('Optional. Resource migration token.')
+param migrationToken string = ''
+
+@description('Optional. Restore a soft-deleted cognitive service at deployment time. Will fail if no such soft-deleted resource exists.')
+param restore bool = false
+
+@description('Optional. Restrict outbound network access')
```
Comment (eriqua) [inline arm/Microsoft.CognitiveServices/accounts/deploy.bicep:127]:
```suggestion
@description('Optional. Restrict outbound network access.')
```

**Review by eriqua — CHANGES_REQUESTED** [review]
Pipeline looks to be failing since the change, template not valid

File: arm/Microsoft.CognitiveServices/accounts/readme.md (hunk 1 above)
Comment (MrMCake) [reply arm/Microsoft.CognitiveServices/accounts/readme.md]: done

**Pull request merged**